gdk/common/exceptions/error_messages.py

```diff
@@ -35,7 +35,10 @@
     "Could not initialize the project as the directory is not empty. Please initialize the project in an empty directory.\nTry"
     " `gdk component init --help`"
 )
-
+INIT_DIR_EXISTS_ERROR = (
+    "Could not initialize the project as the directory already exists. Please initialize the project with a new"
```

Comment:
nit: `Could not initialize the project as the specified directory  <"directory name"> already exists`